lib/simbricks/base/if.c

```diff
@@ -152,6 +152,13 @@ size_t SimbricksBaseIfSHMSize(struct SimbricksBaseIfParams *params) {
 
 int SimbricksBaseIfInit(struct SimbricksBaseIf *base_if,
                         struct SimbricksBaseIfParams *params) {
+  /* a sync interval larger than the latency will lead to incorrect behaviour */
+  if (params->link_latency < params->sync_interval) {
```

Comment:
This should only be checked if synchronization is not disabled through the parameters (optional or forced). If synchronization is disabled neither value matters and can legitimately be configured to 0.